release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1040,6 +1072,34 @@ As part of the continued deprecation of the Operator package manifest format, th
 [id="ocp-4-8-bug-fixes"]
 == Bug fixes
 
+*Builds*
+
+* Previously, after CVE-2021-3344 was fixed, builds did not automatically mount entitlement keys on the {product-title} node. As a result, when the entitlement certificates were stored on the host or node, the fix prevented entitled builds from working seamlessly. The failure to bring in entitlement certificates stored in the host or node was fixed with link:https://bugzilla.redhat.com/show_bug.cgi?id=1945692[BZ#1945692] in 4.7.z and link:https://bugzilla.redhat.com/show_bug.cgi?id=1946363[BZ#1946363] in 4.6.z; however, those fixes introduced a benign warning message for builds running on {op-system-first} worker nodes. The current release fixes this issue by allowing builds to automatically mount entitlements only on {op-system-base} worker nodes, and avoiding mount attempts on {op-system} worker nodes. Now, there will not be any benign warnings around entitlement mounts when running builds on {op-system} nodes.
+ (link:https://bugzilla.redhat.com/show_bug.cgi?id=1951084[*BZ#1951084*])
+
+
+* Some users pulling images from Docker Hub get a `container image registry lookup failed…​toomanyrequests: You have reached your pull rate limit` error. This error happens because the `docker.io` login they used to call the `oc new-app` does not have sufficient paid support with `docker.io`. The resulting application is subject to image pull throttling, which can produce failures.
++
+The current release updates the `oc new-app` help to remind users how defaulting works for the image registry and repository specs, so users can, when possible, use non-default image references to avoid similar errors.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1928850[*BZ#1928850*])
+
+
+* Some users are confused when builds do not apply Docker build arguments in the `buildArgs` array that do not meet the Dockerfile specifications for how `ARG` and `FROM` instructions interact. This release updates the documentation to help users find more information about how Dockerfile `ARG` and `FROM` instructions can interact.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1950653[*BZ#1950653*])
```

Comment:
Do we itemize doc updates in release notes like this?

/cc @rolfedh @Preeticp

Reply:
No, we typically don't mention doc or help content updates like this.

Reply:
Removed. Thanks.